adapters/33across/33acrosstest/exemplary/bidresponse-defaults.json

```diff
@@ -61,7 +61,8 @@
             }
           ],
           "site": {}
-        }
+        },
+        "impIDs":["test-imp-id"]
```

Comment:
I recommend using a formatter on the updates tests. The VS Code json formatter does a good job.

Reply:
Hello @SyntaxNode ,
Is it necessary to apply this modification to all the updated JSON files? Utilizing the JSON formatter seems to impact other content within the file as well. Could we consider creating a separate pull request specifically for this change?

Reply:
We discussed offline and agreed to apply the formatter in a separate PR.